tests/Mapper/ProgrammesDbToDomain/PromotionMapperTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain;
+
+use BBC\ProgrammesPagesService\Domain\Entity\Promotion;
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use BBC\ProgrammesPagesService\Domain\ValueObject\Synopses;
+use BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\PromotionMapper;
+use DateTimeImmutable;
+
+class PromotionMapperTest extends BaseMapperTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->mockMappedImage = $this->createMock('BBC\ProgrammesPagesService\Domain\Entity\Image');
+        $this->mockMappedCoreEntity = $this->createMock('BBC\ProgrammesPagesService\Domain\Entity\Clip');
+    }
+
+    public function testMapperConvertFromDbDataToPromotionWithImage()
+    {
+        $promotionMapper = $this->getPromotionMapper();
+
+        $dbPromotionOfImage = [
+            'id' => 1,
+            'pid' => 'p000000h',
+            'startDate' => new DateTimeImmutable('1900-01-01 00:00:00.000000'),
+            'endDate' => new DateTimeImmutable('3000-01-01 00:00:00.000000'),
+            'weighting' => 73,
+            'isActive' => 1,
+            'promotedFor' => null,
+            'title' => 'active promotion of CoreEntity',
+            'uri' => 'www.myuri.com',
+            'cascadesToDescendants' => false,
+            'createdAt' => new DateTimeImmutable('2017-08-11 10:39:25.000000'),
```

Comment:
You can remove any values that aren't used to create the promotion object, as they're just adding noise.